pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -2350,6 +2359,14 @@ func TestListPlugins(t *testing.T) {
 			},
 			pluginSetCount: 3,
 		},
+		{
+			name: "Add multiple plugins with weights",
+			plugins: &config.Plugins{
+				Score:    config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin, Weight: 3}}},
+				PostBind: config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+			},
+			pluginSetCount: 4,
```

Comment:
`TestListPlugins` is testing the function of `ListPlugins` which is irrelevant with this fix, I'd suggest to create another one to test the change here only.

Reply:
I think we don't need to check the number of plugins as it has already covered by other testcase.